### ISSUE TITLE
Intro program on help (help/index.html) error: "Stage object has no attribute 'SetColor'"

### ISSUE THREAD

**Issue by arloft** [Tymewalk/Slither#96]
Trying to figure out how to use this, and it's not very clear.
Tried the starter program located on index.html in the 'help' folder. Fails with error 'Stage' object has no attribute 'SetColor'.

I've installed slither with pip3. Don't care about SVG at the moment so skipped ImageMagick.
Pasted the code in help/index.html into a .py file, saved it to my desktop, then tried to run it from the terminal with 'python3 start.py' (name of file containing intro code). Fails with error stated above.
Is there a newer intro program? If so, can it be added to the README file, so it's easy to find?

![slither intro program error message](https://user-images.githubusercontent.com/2423302/49548301-3c577880-f8b3-11e8-8ec0-9f4ae59d5e3e.png)

Code taken directly from 'help/index.html':
`import slither
slither.slitherStage.setColor(40, 222, 40)

slither.setup() # Begin slither

def loop():
	pass

slither.runMainLoop(loop)`

**Comment by arloft** [comment]
Removed 'setcolor' from the script and the program ran, opening a white window. 
Was expecting Scratch-like blocks to build programs, disappointed to see it's apparently just a library like pygame. If you guys could build a Scratch-like environment to build programs that are Python programs, and export them to an standalone program, you'd really be on to something.
Maybe I'm missing something, and if so, please tell me. I'm looking for Scratch-like environments that let a user build a program that can be run outside the build environment (if these magical unicorns even exist).

**Comment by BookOwl** [comment]
Yeah, this was meant to be transition library for Scratch programmers who wanted to move to Python but still be able to easily use graphics. Also, I don't think any of us really work on this anymore (I know I haven't in a couple of years).

I think the closest thing to what you want that exists right now is [Snap!](https://snap.berkeley.edu/) and then using [this tool](http://snapp.citilab.eu/) to create a stand alone program. If you have any questions about Snap! @bharvey would be the best one to ask in [the official Snap! forum thread](https://scratch.mit.edu/discuss/topic/4455/). Just tell them BookOwl sent you.

**Comment by BookOwl** [comment]
I'l leave this open because the documentation is wrong and that is a bug.

**Comment by skistaddy** [comment]
It was the initial goal to create the program you are talking about, but I think everyone eventually moved on from slither to better things.
